projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/pojo/AnalysisAnswer.java

```diff
@@ -0,0 +1,78 @@
+package org.batfish.datamodel.pojo;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.batfish.datamodel.answers.Answer;
+
+/**
+ * The {@link AnalysisAnswer AnalysisAnswer} is an Object representation of the analysis answer for
+ * BatFish service.
+ *
+ * <p>Each {@link AnalysisAnswer AnalysisAnswer} contains the corresponding analysis name, an
+ * environment the analysis {@link #_analysisName} were answered, and a list of <@link Answer
+ * answer> as the result.
+ */
+public class AnalysisAnswer {
+  private static final String PROP_ANALYSIS_NAME = "analysisName";
+  private static final String PROP_ENVIRONMENT = "environment";
+  private static final String PROP_ANSWERS = "answers";
+
+  private final String _analysisName;
```

Comment:
Per in-person discussion, the whole object might not be needed. instead, `getAnalysisAnswers(String containerName, Scenario baseScenario, @Nullable Scenario deltaScenario, String analysisName)` can just return `List<Answer>` .. for example.